src/spyglass/data_import/insert_sessions.py

```diff
@@ -101,7 +101,7 @@ def copy_nwb_link_raw_ephys(nwb_file_name, out_nwb_file_name):
     if os.path.exists(out_nwb_file_abs_path):
         if debug_mode:
             return out_nwb_file_abs_path
-        warnings.warn(
+        logger.warn(
```

Comment:
I believe .warn has been deprecated since python 3.3 and the recommendation is to use .warning. See: https://docs.python.org/3/library/logging.html#logging.Logger.warning